pydatastructs/linear_data_structures/tests/test_algorithms.py

```diff
@@ -126,3 +126,36 @@ def test_longest_common_sequence():
     Z = ODA(int, [])
     output = longest_common_subsequence(Y, Z)
     assert str(output) == '[]'
+
+def test_is_sorted():
+    ODA = OneDimensionalArray
+
+    expected_result = True
+
+    arr = ODA(int,[1,2,5,6])
+    output = is_sorted(arr,0,len(arr)-1)
+    assert output == expected_result
```

Comment:
```suggestion
    arr = ODA(int, [1, 2, 5, 6])
    output = is_sorted(arr, 0, len(arr) - 1)
    assert output == expected_result
```
Apply this change whereever necessary